### PR TITLE
Remove useless variable and fix typo

### PR DESCRIPTION
**Pull request by ksaandev**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Remove useless variable from FromController::init()
| Type?         | refacto
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | 

My IDE tells me this variable is unused. I also fixed a typo

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19392)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Comment by matks** [comment]
Hi ! Thank you for the contribution, can you run php-cs-fixer to fix CI ?

**Review by matks — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

File: classes/controller/AdminController.php [hunk 1]
```diff
@@ -2366,8 +2366,6 @@ public function initModal()
             ]);
         }
 
-        // Iso needed to generate Addons login
-        $iso_code_caps = strtoupper($this->context->language->iso_code);
```
Comment (Progi1984) [inline classes/controller/AdminController.php]:
```suggestion
```

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @ksaandev

**Comment by matks** [comment]
No need for QA I think

File: classes/controller/AdminController.php (hunk 1 above)
Comment (matks) [reply classes/controller/AdminController.php]: Thanks for the fix 😉

**Pull request merged**